grails-app/services/org/pih/warehouse/tableroapi/NumberDataService.groovy

```diff
@@ -1,25 +1,27 @@
 package org.pih.warehouse.tableroapi
 
 import org.pih.warehouse.tablero.NumberData
+import org.pih.warehouse.requisition.Requisition
 import org.pih.warehouse.inventory.InventorySnapshot
 
 class NumberDataService {
 
     def dataService
 
-    List<NumberData> getListNumberData(def location){
+    List<NumberData> getListNumberData(def user, def location){
```

Comment:
Minor comment. Try to use a type where possible instead of def. In this case User and Location, respectively.